zh_cn/docs/开发者/vx.json

```diff
@@ -13,6 +13,9 @@
         }
     ],
     "folders": [
+        {
+            "name": "开发入门"
```

Comment:
Unnecessary to create a new folder. Just put it under the parent folder.

Reply:
I'm going to write a series of documents.